ajax/libs/WhitestormJS/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "WhitestormJS",
```

Comment:
The name in package.json from upstream is `whs`. I think we can just follow it. Ref: https://github.com/WhitestormJS/whitestorm.js/blob/beta/package.json#L2